arboretum/common/iam_ibm.py

```diff
@@ -0,0 +1,22 @@
+"""Utility function for IBM Cloud IAM."""
+import requests
+
+
+def get_tokens(api_key):
+    """Get tokens (access token and refresh token) by api_key.
```

Comment:
```
    """
    Get tokens (access token and refresh token) by api_key.
```

Reply:
- If the point of this function is to return a token then why not have this function return just that rather than the entire response content?
- I don't see this function being used anywhere else.  Is the plan for it to be used in later fetchers?  
- I also don't see any refresh token usage.  Is there actually a "refresh_token" that is different from the "access_token"?  If so then maybe make this function return one or the other based on an input parameter.  Or perhaps have it return both as a tuple.
- I'd also change the name to `get_token` if it is going to return one token.

Reply:
Now `get_tokens()` just returns a tuple of `access_token` and `refresh_token`.
`refresh_token()` is not used by the code of this PR, but it will be used by cluster resource fetcher (e.g., [to get config for cluster](https://github.com/ComplianceAsCode/auditree-arboretum/blob/2479a843e6a837f5451580282493b8da10999169/arboretum/kubernetes/fetchers/fetch_cluster_resource_ibm_cloud.py#L44)).